openedx/core/lib/xblock_fields/fields.py

```diff
@@ -253,3 +253,38 @@ def enforce_type(self, value):
             return value
 
         return self.from_json(value)
+
+
+class TimeInfo(object):
```

Comment:
I'm not looking at this, and assuming it was a direct move.  Let me know if that is not the case.